### DISCUSSION TITLE
first layer little is printed a little bigger...

### DISCUSSION THREAD

**Discussion by Mario-vminion** [OrcaSlicer/OrcaSlicer#181] [Q&A]
hi when i slice model print it without a brim of else then i see that the first layer is a little bigger .. like you have when you take away the brim there is than also a small line next to the print..
i have treed to do the line width first layer .4 it was standard .45 but no effect...
so maybe someone has the same issue

**Comment by djngnear** [comment]
Adjust the elephant foot compensation under Quality-Precision

**Reply by Mario-vminion** [reply]
thanks it worked...
